virus/obeliscus_porcinus.md

```diff
@@ -0,0 +1,314 @@
+# {Pigs Might Fly: Exploring the Potential of Obeliscus porcinus omega36 in the Antisense Protein Coding}
+written by: [David Zheglov](https://github.com/dzheglov)
+
+### {Obeliscus porcinus Ω36}
+*{ The name Obeliscus refers to the overarching family of obelisk RNAs, characterized by their rod-like secondary structure. Porcinus is derived from Latin, meaning "of pigs," reflecting its ecological association with the porcine microbiota. The designation omega36 denotes the cluster number assigned during classification. I will also use the name omega36 throughout the text, because it is easier to refer to the obelisk with a short name.}*
+
+[Optional: Tutorial 2 Generative AI Cover-art](img/O_porcinus_omega36/00_cover_art.jpg)
+
+## Abstract
+
+RNA viroid-like elements of the obelisk family are a recently discovered group of genetic entities with circular genomes and unique secondary structures. These RNAs are thought to colonize various ecosystems and exhibit remarkable versatility in protein-coding potential. Here, we describe Obeliscus porcinus omega36, a novel member of the obelisk family. Omega36 is distinct due to its genome length of 1586 nucleotides, larger than previously characterized members such as Obeliscus alpha. This genome encodes two well-characterized proteins, Oblin-1 and Oblin-2, along with a potential third protein, Oblin-3, located on the antisense strand.
+
+In our analysis, we classified the genomes of the first 100 obelisk clusters into distinct groups based on their coding potential and genomic architecture. While many obelisks encode only one or two proteins, some, like cluster 36, may encode a third protein on the antisense strand. Others, such as cluster 33, exhibit evidence of recent genome duplication, producing a symmetrical genome that encodes identical proteins on the sense and antisense strands.  These results indicate that there is a high degree of variability in the evolution of the obelisk family and may suggest the potential for new ways of protein coding in these RNA elements.
+
+The ecological distribution of omega36 also strengthens the percription of omega36 as an obelisk RNA. Thus, omega36 was detected in geographically different datasets originated from China and the USA and was linked with porcine microbiome and bacterial hosts.
+
+## Results
+
+### Ecology subsection A...
```

Comment:
Amend out the "Default Titles" from the document. Your descriptive titles are much better throughout.